### PR TITLE
Apply session icon

### LINKED ISSUE #62: [Assigned] Apply session icon
Overview
Apply feed tab design.
Session icons for each level icons.
And that design apply

### PR DESCRIPTION
**Pull request by takahirom**
close #62

## Overview (Required)
- Apply design

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
Commented. Almost LGTM! 👍

File: app/src/main/res/layout/item_speech_session.xml [hunk 1]
```diff
@@ -146,18 +124,28 @@
...
-                android:layout_marginTop="16dp"
+                android:layout_marginStart="8dp"
                 android:maxLines="2"
                 android:visibility="@{simplify ? View.GONE : View.VISIBLE}"
+                app:layout_constraintBottom_toBottomOf="@id/level"
                 app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="@id/title"
-                app:layout_constraintTop_toBottomOf="@id/description"
+                app:layout_constraintStart_toEndOf="@id/level"
+                app:layout_constraintTop_toTopOf="@id/level"
```
[the remark this answers is not in the document]
Comment (takahirom) [reply app/src/main/res/layout/item_speech_session.xml]: Probably it comment is fixed by this https://github.com/DroidKaigi/conference-app-2018/pull/396/commits/d7232d1bb5abda154b1182cd992d79a93dc8f7c3

[the remark this answers is not in the document]
Comment (konifar) [reply app/src/main/res/layout/item_speech_session.xml]: Perfect 💯 Thanks for your quick fix!

**Comment by takahirom** [comment]
Thanks 👍

**Pull request merged**